_data/cloud_releases.csv

```diff
@@ -44,3 +44,4 @@ date,sha
 2023-02-09,null
 2023-03-06,e36550014fba4431878d5280c55480fb067e576c
 2023-03-31,null
+2023-04-10,HEAD
```

Comment:
Is this correct? Nearly all the other entries have `null` for the last field. Should this be a hash?

Reply:
It should be a hash, I think it didn't work with the null entry before but not sure. will look into later this week.